Makefile

```diff
@@ -14,6 +14,10 @@ MOLECULE_CONFIG ?= ${MOLECULE_CONFIG:-.config/molecule/config_podman.yml}
 # Run molecule against all tests
 TEST_ALL_ROLES ?= ${TEST_ALL_ROLES:-no}
 
+CIFMW_LOCAL_ENV_IMAGE_URL ?= https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Stream-GenericCloud-9-20230327.0.x86_64.qcow2
+
+CIFMW_LOCAL_ENV_BASE_DIR ?= /opt/ci-framework
```

Comment:
Since the user is going to run make local_env_create from ci_framework. So can we change like this.
```
CIFMW_LOCAL_ENV_BASE_DIR ?= ${PWD}
```

Reply:
Sure? As we are gonna run as root we can end with root file under users $HOME, seems ugly...

Reply:
We shouldn't use root nor `sudo` - really. Think "user", and if really, requires the `sudo` only for the needed pieces. This is mostly addressed in the ansible roles actually. One more reason to consume them :)

Reply:
new role takes this comment into account.